taskt/UI/Forms/frmScriptEngine.cs

```diff
@@ -25,14 +25,15 @@
 using taskt.Core;
 using taskt.Core.Automation.Engine;
 using static taskt.Core.Automation.User32.User32Functions;
+using taskt.Core.Automation.Engine.EngineEventArgs;
 
 namespace taskt.UI.Forms
 {
     public partial class frmScriptEngine : ThemedForm
     {
         //all variables used by this form
         #region Form Variables
-        public Core.EngineSettings engineSettings;
+        private Core.EngineSettings _engineSettings;
         public string filePath { get; set; }
         public string xmlData { get; set; }
         public Core.Server.Task remoteTask { get; set; }
```

Comment:
### Variable/Property Declaration Convention:

I think you forgot to apply **PascalCase** for all public and **_camelCase** for all negative variables/properties.
Please apply through out the PR.